src/rich-text-editor.component.js

```diff
@@ -228,8 +228,10 @@ export const RichTextEditor = forwardRef((props, editorRef) => {
   }
 
   function getBrowserDefaultStyling() {
-    if (props.placeholderColor) {
-      return `color: ${props.placeholderColor};`
+    if (props.placeholderStyle) {
+      return Object.entries(props.placeholderStyle).reduce((styleString, [propName, propValue]) => {
```

Comment:
This doesn't allow for all caps css rules which is actually something allowed by the css specification. I messaged you on slack about this - it might be fine to keep it but let's discuss options.